shared/common-adapters/index.js

```diff
@@ -101,6 +101,7 @@ export {default as StandardScreen} from './standard-screen'
 export {default as TabBar} from './tab-bar'
 export {default as Tabs} from './tabs'
 export {default as Text} from './text'
+export {default as TimelineMarker} from './timeline-marker'
```

Comment:
We should TODO migrating devices to use this, I didn't want to touch that in this PR